djangocms_navigation/admin.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
+from treebeard.admin import TreeAdmin
+from treebeard.forms import movenodeform_factory
+
+from .models import Menu, MenuContent, MenuItem
+
+
+class MenuContentAdmin(admin.ModelAdmin):
+    exclude = ['modified_by', 'menu', ]
+
+    def save_model(self, request, obj, form, change):
+        if not obj.pk:
+            obj.menu = Menu.objects.create()
+        obj.modified_by = request.user
+        super().save_model(request, obj, form, change)
+
+
+class MenuItemAdmin(TreeAdmin):
+    form = movenodeform_factory(MenuItem)
+
+    exclude = ['modified_by', ]
+
+    def save_model(self, request, obj, form, change):
+        obj.modified_by = request.user
```

Comment:
As discussed, I think either maybe we want to move modified_by to versioning or remove it (if it's not possible to ever have somebody else modify a version other than the person who created it) or do `obj.menu_content.modified_by = request.user`